CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.100.3 (2019-12-03)
+
+* The `aspectRatio` option to the `attachments` schema field type is now fully implemented. We always had this for selecting images, e.g. in our `apostrophe-images-widgets` module, but it is now also available when directly using an `attachment` schema field as a property of your own doc. You can also set `crop: true` to allow manual cropping in that case. This is a useful technique when including the image in a reusable media library does not make sense.
```

Comment:
The description here implies that you can use `aspectRatio` independently from `crop`. But if I only set the `aspectRatio` property it doesn't auto crop until I set `crop: true` as well. That makes sense with the conditional written on line 187 of user.js.

So we should either update this or edit the conditional to allow auto cropping. I gather the latter.

Reply:
Good point. I think aspectRatio should full-on imply crop because there isn't anything to be gained by denying the user a little more control over what must happen anyway.